src/truncate/truncate.component.js

```diff
@@ -0,0 +1,28 @@
+angular.module('patternfly.truncate').component('pfTruncate', {
+
+  bindings: {
+    content: '<',
+    limit: '<',
+    useWordBoundary: '<',
+    newlineLimit: '<',
+    removeExtraChars: '<',
+    expandCopy: '<',
+    collapseCopy: '<',
+    // When expandable is on, optionally hide the collapse link so text can only be expanded. (Used for toast notifications.)
+    expandable: '<',
+    //hideCollapse: '<',
+    // optional keywords to highlight using the `highlightKeywords` filter
+    keywords: '<',
+    prettifyJson: '<'
```

Comment:
It was part of the original port from [Origin-Web-Common](https://github.com/cdcabrera/origin-web-common/blob/master/src/components/truncate-long-text/truncateLongText.js#L18), currently it doesn't do anything. Was debating on whether to even include it or come up with some form of transclusion for that type of content...